docs/admin/upgrade/planning.rst

```diff
@@ -5,7 +5,7 @@
 
 
 ==========================
-General upgrade guidelines
+General Upgrade Guidelines
```

Comment:
Any idea to make this shorter?

Reply:
Maybe just use the correct label to reflect "What's Inside", instead of convoluting it?

```suggestion
Upgrade Planning
```

Reply:
just `Guidelines` ?

Reply:
Upgrade Guidelines?

Reply:
Ah, "Upgrade Planning" is only a subsection of that page, I am just seeing it 🤦. Sure, why not.

Reply:
I've adjusted the patch, see https://github.com/crate/cratedb-guide/pull/43#pullrequestreview-1903835131.